packages/theming/default-theme/package.json

```diff
@@ -41,6 +41,7 @@
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
     "@fluentui-react-native/scripts": "^0.1.1",
+    "@office-iss/react-native-win32": "^0.68.0",
```

Comment:
I'd also add it peer dependencies while we're at it.

Reply:
In all of the package.json files that include this, it's only under devDependencies. Should we also be adding to to peer dependencies for all of the package.json files?

Reply:
Oh, if that's the case then nvm. IMO yes anything that takes this fork as a devDependency should also take it as a peer dependency. But it's not a big deal.